english/candyland/diabetes.md

```diff
@@ -0,0 +1,20 @@
+You light a small fire in the middle of the marshmallow room.
+
+As the fire grows, a sweet aroma fills the room and the ceiling becomes ever so slightly brown just above the fire.
+
+The marshmallow room started melting and you started seeing some colorful lights shining through.
+
+Wait! But this is not any colorful lights you are seeing!  What appears before you is a rainbow colored Gumdrop Mountains!
+
+Surrounding the Gumdrop Mountain are acres of Lollipop Woods where there's a forest of giant lollipops as tall as sequoia.
+
+You wanted to taste all these sweetness around you, but damn you just realize you have diabetes.
+
+
+[Light fire] (light-fire/fire.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.